components/brave_wallet/browser/swap_request_helper.cc

```diff
@@ -132,4 +136,287 @@ std::optional<std::string> EncodeJupiterTransactionParams(
   return result;
 }
 
+}  // namespace jupiter
+
+namespace lifi {
+
+namespace {
+
+std::optional<std::string> EncodeChainId(const std::string& value) {
+  if (value == mojom::kSolanaMainnet) {
+    return "SOL";
+  }
+
+  uint256_t val;
+  if (!HexValueToUint256(value, &val)) {
+    return std::nullopt;
+  }
+
+  if (val > std::numeric_limits<uint64_t>::max() ||
+      val < std::numeric_limits<uint64_t>::min()) {
```

Comment:
val is always positive, so second comparison is not needed

Reply:
13b492b4b105a1745680932e3c1933133db8ab11